src/schema/auth.ts

```diff
@@ -4,29 +4,12 @@ import { Endpoint } from "../helpers";
 import { privateUser } from "./types";
 
 export const authEndpoints: Record<string, Endpoint> = {
-    get_token: {
```

Comment:
I get that you want to do auth via HTTP, but I would still like to have the schema for that defined somewhere. I assume there's also gonna be a need for other unauth'd endpoints via HTTP, e.g. account recovery, so maybe we could have a dedicated file just for those schemas?

Reply:
I'd be happy to do that. I'll add a file for it later today/tomorrow.